### DISCUSSION TITLE
Ability to execute Ruby code?

### DISCUSSION THREAD

**Discussion by nirvdrum** [quarto-dev/quarto-cli#5336] [Q&A] [answered]
The built-in support for Python is amazing. In my day-to-day, I'm working with Ruby and would love to be able to evaluate Ruby snippets in Quarto. I realize there's no first class functionality for Ruby, but is there a mechanism for executing arbitrary commands and capturing standard output?

**Comment by mcanouil** [comment, marked as answer]
Both `knitr`/`Jupyter` could run Ruby code as long as you set all the dependencies and set your document/project with the proper information, see <https://quarto.org/docs/computations/execution-options.html#engine-binding>.

**Reply by nirvdrum** [reply]
Thanks. I started looking at Quarto for an upcoming presentation. The sample presentation in VS Code shows Python but I couldn't find anything on using a different language. I see now that all of Quarto is available and I should look beyond just the presentation docs.
